app/bootstrap.php

```diff
@@ -8,7 +8,9 @@
  * Environment initialization
  */
 error_reporting(E_ALL);
-stream_wrapper_unregister('phar');
+if (!in_array('phar', \stream_get_wrappers())) {
```

Comment:
I believe there should be no `!`: https://3v4l.org/Je0Q5

After changes are applied and all builds are green, please squash them into a single commit so that we have perfectly clean history 😉

Reply:
Oops , commit was change.
Thanks